src/backend/cassandra/impl/Statement.h

```diff
@@ -91,6 +92,8 @@ class Statement : public ManagedObject<CassStatement>
         using decayed_t = std::decay_t<Type>;
         using uchar_vec_t = std::vector<unsigned char>;
         using uint_tuple_t = std::tuple<uint32_t, uint32_t>;
+        using uint_bytes_tuple_t = std::tuple<uint32_t, ripple::uint256>;
+        using bytes_vector_t = std::vector<ripple::uint256>;
```

Comment:
@godexsoft this is where i could use your help with better application of generics. i think it's possible to get tuples abstracted better

Reply:
Yes. Let's discuss over a call later.